### DISCUSSION TITLE
sticky parallax

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#2015] [Q&A]
Not both together (I don't think), but has anyone seen the ability to do a parallax background, if even by just adding a Vanjaro-specific style?

Sort of the same question for making a Section sticky, moistly to the top for a nav strip.

**Comment by Tychodewaard** [comment]
Parallax background: is that 
![image](https://user-images.githubusercontent.com/9091997/148250366-06f7b831-6549-4c8c-8841-c8095dc2140b.png)
?

**Reply by eswanzey** [reply]
totallymissedit, thanks @tychodewaard!

**Comment by Tychodewaard** [comment]
The sticky requires a bit of javascript. And classes that refer to that script. It is not included in Vanjaro. At least not in the free version. I think there are themes in the commercial version that support this out of the box. But @meetmandeep knows this better than me :-)

**Reply by meetmandeep** [reply]
Yes. Sticky header is available in Premium Theme here:

![image](https://user-images.githubusercontent.com/29958123/148277586-143c62b6-334a-4e2e-8bb0-59d834d4c81f.png)
